yt_dlp/extractor/cbs.py

```diff
@@ -97,6 +146,11 @@ def _extract_video_info(self, content_id, site='cbs', mpx_acc=2198311517):
                     update_url_query(tp_release_url, query), content_id,
                     'Downloading %s SMIL data' % asset_type)
             except ExtractorError as e:
+                if not useXMLmetadata:
+                    query['formats'] = formatList[1]
+                    tp_formats, tp_subtitles = self._extract_theplatform_smil(
+                        update_url_query(tp_release_url, query), content_id,
+                        'Downloading %s SMIL data, trying again with another format' % asset_type)
                 last_e = e
                 continue
```

Comment:
This should be removed, no?

Reply:
Yeah, it really serves no purpose as the video can't be downloaded anyway if formats = `'M3U+none,MPEG4,M3U+appleHlsEncryption,MP3'` doesn't yield anything from theplatform's API. Only used to tell if the video expired, but it's not really important. Do you want me to remove the entire formatList / `if not useXMLmetadata:` section to simplify things?